push_pattern_test.go

```diff
@@ -1,361 +0,0 @@
-package main
-
-import (
-	"testing"
-
-	"github.com/phrase/phraseapp-client/internal/paths"
-)
-
-type Patterns []*Pattern
-
-type Pattern struct {
-	File         string
-	Ext          string
-	TestPath     string
-	ExpectedCode string
-	ExpectedName string
-	ExpectedTag  string
-}
-
-func TestSpecialCharacters(t *testing.T) {
-	patterns := Patterns{
-		{
-			File:     "./locales/?*.yml",
-			Ext:      "yml",
-			TestPath: "locales/?en.yml",
-		},
-		{
-			File:         "./abc+/defg./}{x/][etc??/<locale_code>.yml",
-			Ext:          "yml",
-			TestPath:     "abc+/defg./}{x/][etc??/en.yml",
-			ExpectedCode: "en",
-		},
-	}
-	patterns.TestPatterns(t)
-}
-
-func TestSinglePlaceholders(t *testing.T) {
-	patterns := Patterns{
-		{
-			File:         "./.abc/<locale_code>.yml",
-			Ext:          "yml",
-			TestPath:     ".abc/en.yml",
-			ExpectedCode: "en",
-		},
-		{
-			File:     "./*.yml",
-			Ext:      "yml",
-			TestPath: "en.yml",
-		},
-		{
-			File:     "./locales/en.yml",
-			Ext:      "yml",
-			TestPath: "locales/en.yml",
-		},
-		{
-			File:     "./locales/.yml",
-			Ext:      "yml",
-			TestPath: "locales/en.yml",
-		},
-		{
-			File:         "./abc/defg/<locale_code>.lproj/.strings",
-			Ext:          "strings",
-			TestPath:     "abc/defg/en.lproj/Localizable.strings",
-			ExpectedCode: "en",
-		},
-		{
-			File:         "./locales/<locale_code>.yml",
-			Ext:          "yml",
-			TestPath:     "locales/en.yml",
-			ExpectedCode: "en",
-		},
-		{
-			File:         "./abc/<locale_code>.lproj/.strings",
-			Ext:          "strings",
-			TestPath:     "abc/en.lproj/Localizable.strings",
-			ExpectedCode: "en",
-		},
-	}
-	patterns.TestPatterns(t)
-}
-
-func TestMultiplePlaceholders(t *testing.T) {
```

Comment:
@alexanderwilling (Question/blocker) Where are these tests now? I cant find them .. These tests are not arbitrary, they are actual patterns from users. I cant seem to find them in the "new" test suite. Can you point me to the location? (e.g. they are a standard that must pass)